model_compression_toolkit/core/common/quantization/quantization_params_generation/power_of_two_selection.py

```diff
@@ -54,7 +55,7 @@ def power_of_two_selection_tensor(tensor_data: np.ndarray,
         tensor_max = get_tensor_max(tensor_data, per_channel, channel_axis)
         threshold = max_power_of_two(tensor_max, min_threshold)
     else:
-        signed = np.any(tensor_data < 0)
+        signed = True  # weights are always signed
```

Comment:
Please also check symmetric quantization

Reply:
isn't power of 2 always symmetric?